### DISCUSSION TITLE
Error with restart when running autotrain

### DISCUSSION THREAD

**Discussion by razvanab** [huggingface/autotrain-advanced#477] [Q&A]
I didn't want to post this in issues since I don't know exactly how to report it.

I use Windows 11 with Conda and a Python 3.11 environment created for Autotrain.

When I run "autotrain app --host 0.0.0.0" the command is restarting my PC with a non-error, exactly after the yellow text is shown in the console.

**Comment by abhishekkrthakur** [comment]
is it wsl?

**Comment by razvanab** [comment]
No wsl

Here is dxdiag maybe this helps somehow.
[ https://pastebin.com/RhA2YwA2 ](url)

**Reply by abhishekkrthakur** [reply]
windows support is not fully possible atm. WSL _should_ work. there are people who have made it work on windows but you'll have to scan through the closed issues. bitsandbytes still doesnt support windows so many parts such as peft wont work. the other option is to use the docker image `huggingface/autotrain-advanced:latest`

**Comment by razvanab** [comment]
I see.
Thank you!
